flexget/utils/imdb.py

```diff
@@ -166,7 +168,15 @@ def search(self, name):
             movie = {}
             additional = re.findall(r'\((.*?)\)', row.text)
             if len(additional) > 0:
-                movie['year'] = additional[-1]
+                try:
+                    int(additional[-1])
```

Comment:
Im not a fan of it either, if you know a better way to do it, please, advise. This is the google result for "python isint".

http://stackoverflow.com/questions/1265665/python-check-if-a-string-represents-an-int-without-using-try-except

Should I use .isdigit() or regex matching?